BotRoyaleUnity/Assets/Scripts/CollisionDetection.cs

```diff
@@ -24,10 +26,20 @@ void OnCollisionEnter(Collision collision)
             Collider other = contact.otherCollider;
             if (!other.CompareTag("Untagged"))
             {
-                //Debug.Log(current.tag + " "+ current.name + " : " + other.tag + " " + other.name);
-                if (current.CompareTag("Spike"))
+                if (!other.CompareTag("Wall"))
```

Comment:
Not sure if we need to be checking what the other object's tag isnt, we just need to know if it has a PartHealth, if `other.gameObject.GetComponent<PartHealth>() != null`